test/scales/tickGeneratorsTests.ts

```diff
@@ -22,35 +23,62 @@ describe("Scales", () => {
       });
 
       it("generates ticks for a domain crossing 0", () => {
-        let start = -1.5, end = 1, interval = 0.5;
+        let start = -1.5;
+        let end = 1;
         scale.domain([start, end]);
-        let ticks = Plottable.Scales.TickGenerators.intervalTickGenerator(interval)(scale);
+        let ticks = Plottable.Scales.TickGenerators.intervalTickGenerator(0.5)(scale);
```

Comment:
Not necessarily one interval value for the entire block, but I would like to place these magic numbers outside in a variable in case we want to tweak these numbers when we modify the tests later.